app/uk/gov/hmrc/lisaapi/connectors/DesConnector.scala

```diff
@@ -162,7 +162,7 @@ class DesConnector @Inject()(
   }
 
   /**
-    * Attempts to close a LISA account
+    * Attempts to report a LISA account
```

Comment:
Change from close to report?
Please double check the method is closeAccount

Reply:
Don't recall making this change - must have slipped in. I'll revert it back to close.